src/App/App.js

```diff
@@ -18,26 +18,40 @@ class App extends Component {
     this.setState({ [key]: event.target.value })
   }
 
-  submit = () => {
+  loginUser = (event) => {
+    event.preventDefault();
     if (this.state.name === '' || this.state.email === '' || this.state.type === '') {
       document.querySelector('.error').classList.remove('hidden');
     } else {
       this.props.history.push('/areas')
     }
   }
 
+  logoutUser = (event) => {
+    event.preventDefault();
+    this.state.name = ''
+    this.state.email = ''
+    this.state.type = ''
+    this.props.history.replace('/')
+  }
+
```

Comment:
So smart to use state to control this. Should it be this.setState or can you change it this way too?

Reply:
I should probably use setState.  Will update now.